packages/cli/src/runners/__tests__/task-broker.test.ts

```diff
@@ -60,13 +61,13 @@ describe('TaskBroker', () => {
 			taskBroker.registerRunner(runner, messageCallback);
 
 			const knownRunners = taskBroker.getKnownRunners();
-			const runnerIds = Object.keys(knownRunners);
+			const runnerIds = [...knownRunners.keys()];
 
 			expect(runnerIds).toHaveLength(1);
 			expect(runnerIds[0]).toEqual(runnerId);
 
-			expect(knownRunners[runnerId].runner).toEqual(runner);
-			expect(knownRunners[runnerId].messageCallback).toEqual(messageCallback);
+			expect(knownRunners.get(runnerId)!.runner).toEqual(runner);
+			expect(knownRunners.get(runnerId)!.messageCallback).toEqual(messageCallback);
```

Comment:
nit: Could use `?` here instead of `!`. In that case we get `AssertionError` from jest with proper formatting, where as with `!` we get `ReferenceError` from runtime if the `knownRunners` don't have `runnerId`